src/pages/Main/Main.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import { InputField, Results } from "components";
+import "./Main.css";
+
```

Comment:
I added a typescript `Interface` here for our `state`. Have a look and then check out what I did in `InputField`, and then apply the same thing for `props` in `Results`.